pkg/apis/api.acorn.io/v1/scheme.go

```diff
@@ -57,6 +57,12 @@ func AddToSchemeWithGV(scheme *runtime.Scheme, schemeGroupVersion schema.GroupVe
 		&AcornImageBuildList{},
 		&ComputeClass{},
 		&ComputeClassList{},
+		&ImageAllowRules{},
+		&ImageAllowRulesList{},
+		&ProjectImageAllowRules{},
+		&ProjectImageAllowRulesList{},
+		&ClusterImageAllowRules{},
+		&ClusterImageAllowRulesList{},
```

Comment:
I'm slightly confused by these types. When we did this pattern for `ComputeClasses` and `VolumeClasses` it made sense as the `ImageAllowRules` (in this example) served as an aggregate user facing type for the `Project*` and `Cluster*` admin  facing types. I notice that there is no way for a user to query the currently available `ImageAllowRules` via the CLI so could you help understand the need for the `ImageAllowRules` type?

Reply:
Yes, `ImageAllowRules` is an aggregate, and listing it as a combined list of cluster and project rules, it also works for me :thinking:

Reply:
As per latest alignment, we won't have Acorn CLI functions for this right now.
Also, the distinction between Cluster* and Project* will be removed now.